containers/README.md

```diff
@@ -6,7 +6,7 @@ Container image gets automatically built by quay.io at [Kraken image](https://qu
 
 ### Run containerized version
 
-Refer [instructions](https://github.com/redhat-chaos/krkn/blob/main/docs/installation.md#run-containerized-version) for information on how to run the containerized version of kraken.
+Refer [instructions]([https://github.com/redhat-chaos/krkn/blob/main/docs/installation.md#run-containerized-version](https://krkn-chaos.dev/docs/installation/)) for information on how to run the containerized version of kraken.
```

Comment:
think there's a link in a link here. I'm not seeing it render correctly here: https://github.com/NarmalaSk/krkn/blob/patch-1/containers/README.md
Think it should just be this: 
`[instructions](https://krkn-chaos.dev/docs/installation/)`